buildlib/azure-pipelines-release.yml

```diff
@@ -55,6 +55,8 @@ stages:
             tag: $(Build.SourceBranchName)
             isDraft: true
             addChangeLog: false
+            releaseNotesSource: input
+            releaseNotes: (release notes going here)
```

Comment:
is it a new requirement from Azure?
can you pls PR this to v1.9 branch as well?

Reply:
@yosefe Good question.
Probably, MS has added "ReleaseNotes*" attribute to the "GithubRelease" task recently. And it tried to use `path: "we/need/to/go/deeper"`  from the checkout step step above. We (Andrey and me) Added "we/need/to/go/deeper" path 1 months agor to overcome rpmbuild failures in a docker container.

Let me check this change on some hello-world project before merging (please add WIP label to this PR).

Reply:
Well, `releaseNotesSource` defaults to `file` and `releaseNotesFile` was not specified in our case.
So that I decide to add `ReleaseNoted.md` into the root of source tree, point at it from the pipeline file.

There is an example of test run:
   https://github.com/yshestakov/ucx/releases

@shamisp @yosefe Are you agree to have `ReleaseNotes.md` file in the source tree and put proper content there before release (before triggering AZP release job)?

Reply:
Can we redirect the pointer to NEWS, which is effectively our releasenotes.md ? Or the file has to be named releasenotes.md ?

Reply:
@shamisp  of course, I can change pointer to NEWS.
However, it contains the whole history of releases. Do you this history to be posted on the particular release page?
![ucx-release-draft](https://user-images.githubusercontent.com/165710/89980556-cce67e80-dc7a-11ea-9aa6-e7f3d94a9716.jpg)

Reply:
probably not want this
let's start with stub and update news manually for now
@yshestakov so this PR will fix release issue?
do we really need to add azure-pipelines-release-yucx.yml ? it seems a duplication of azure-pipelines-release.yml

Reply:
@yosefe 
  1) yes, this PR fixes the issue with "Not Found" (releaseNotes file)
  2) yep, no need to add `add azure-pipelines-release-yucx.yml`, removed

Reply:
I don't like that overwrites release notes but I guess I can re-open and edit those ?

Reply:
@shamisp well, the job just creates a draft (of release). You can modify if needed.
On another hand, I can set `releaseNoteSource: input` and put some inline text like "TBD" into the pipeline.
In such case you don't need to maintain`ReleaseNotes.md` in the git repo.